build/Dockerfile

```diff
@@ -138,9 +137,10 @@ COPY --chown=nginx:0 LICENSE /licenses
 
 ############################################# Base image for Openshift OSS #############################################
 FROM openshift-base AS openshift
-ARG NGINX_VERSION
 ARG IC_VERSION
 
+ENV NGINX_VERSION 1.19.9
```

Comment:
Unfortunately, the base image for openshift is not specific for NGINX and doesn't have the variable so we need to set it manually for now.